tailwind.config.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
       events: "url('/assets/backgrounds/events.jpg')",
       contact: "url('/assets/backgrounds/contact.jpg')",
       aksaraBox: "url('/assets/backgrounds/aksaraBox.png')",
+      winston: "url('/assets/backgrounds/winston.jpg')",
```

Comment:
Can try to explain whats the purpose of putting a path in `tailwind.config.js` here